backend/dal/Data/Entities/Address.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 
-namespace Pims.Api.Data.Entities
+namespace Pims.Dal.Data.Entities
```

Comment:
I think I'd also extract all the `Entities` into their own project `Pims.Dal.Entities`.  This way we can at some point if required fully control access to the database through the `Dal`, but also share the `Entities` to other projects without the need to reference the `Dal`.